README.md

```diff
@@ -1,5 +1,137 @@
+
+The code is licensed under an Apache 2.0 license.
+
+See [the license file](LICENSE) for further details.
+
+## Platform support
+
+Kata Containers currently works on systems supporting the following
+technologies:
+
+- [Intel](https://www.intel.com)'s VT-x technology.
+- [ARM](https://www.arm.com)'s Hyp mode (virtualization extension).
+
+### Hardware requirements
+
+The runtime has a built-in command to determine if your host system is capable
+of running a Kata Container:
+
+```bash
+$ kata-runtime kata-check
+```
+
+> **Note:**
+> 
+> If you run the command above as the `root` user, further checks will be
+> performed (e.g. it will check if another incompatible hypervisor is running):
+> 
+> ```bash
+> $ sudo kata-runtime kata-check
+> ```
+
+## Quick start for developers
+
+See the
+[developer guide](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md).
+
+## Configuration
+
+The runtime uses a TOML format configuration file called `configuration.toml`.
+The file contains comments explaining all options.
+
+> **Note:**
+> 
+> Most users will not need to modify the default configuration.
+
+Since the runtime supports a
+[stateless system](https://clearlinux.org/features/stateless),
+it checks for this configuration file in multiple locations, two of which are
+built in to the runtime. The default location is
+`/usr/share/defaults/kata-containers/configuration.toml` for a standard
+system. However, if `/etc/kata-containers/configuration.toml` exists, this
+will take priority.
+
+The command below will list the full paths to the configuration files the runtime
+will attempt to load. The first path that exists will be used:
+
+```bash
+$ kata-runtime --kata-show-default-config-paths
+```
+
+Aside from the built-in locations, it is possible to
+specify the path to a custom configuration file using the `` option:
+
+```bash
+$ kata-runtime --kata-config=/some/where/configuration.toml
+```
+
+The runtime will log the full path to the configuration file it is using. See
+the [logging](#Logging) section for further details.
+
+To see details of your systems runtime environment (including the location of
+the configuration file being used), run:
+
+```bash
+$ kata-runtime kata-env
+```
+
+## Logging
+
+The runtime provides `--log=` and `--log-format=` options. However, the
+runtime will always log to the system log (syslog or `journald`).
+
+To view runtime log output:
+
+```bash
+$ sudo journalctl -t kata-runtime
+```
+
+For more thorough analysis and details of how to obtain logs for other system components, see the documentation for the
+[kata-log-parser](https://github.com/kata-containers/tests/tree/master/cmd/log-parser)
+tool.
+
+## Debugging
+
+See the
+[debugging section of the developer guide](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#enable-full-debug).
+
+## Community
+
+See [the community repository](https://github.com/kata-containers/community).
+
+## Home Page
```

Comment:
I would maybe put this at the top, not the bottom?

Reply:
Reworked to remove that section entirely and put the links at the top.